Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -351,6 +351,7 @@ import BraintreeDataCollector
 
         var urlComponents = URLComponents(url: payPalAppRedirectURL, resolvingAgainstBaseURL: true)
         urlComponents?.queryItems = [
+            URLQueryItem(name: "ba_token", value: payPalContextID),
```

Comment:
This paypalContextID could be either a ba_token or ec_token, based on [this logic](https://github.com/braintree/braintree_ios/blob/761fdce14760455129c864a7970d48989a80226c/Sources/BraintreePayPal/BTPayPalApprovalURLParser.swift#L22-L37).

We should ensure that we only proceed if a `ba_token` is present

Reply:
DTBTSDK-3781